lib/smb2/packet.rb

```diff
@@ -47,17 +47,134 @@ class InvalidFlagError < StandardError; end
   # Constants
   ##
 
-  # Values for {QueryInfoRequest#info_type}
-  # @see https://msdn.microsoft.com/en-us/library/cc246557.aspx
-  QUERY_INFO_TYPES = {
-    # SMB2_0_INFO_FILE
-    FILE: 0x01,
-    # SMB2_0_INFO_FILESYSTEM
-    FILESYSTEM: 0x02,
-    # SMB2_0_INFO_SECURITY
-    SECURITY: 0x03,
-    # SMB2_0_INFO_QUOTA
-    QUOTA: 0x04
+  # Used in {CreateRequest#disposition}
```

Comment:
Could you add `@note Ordered by key value` so that I don't think they're out of order and so that other developers understand where to insert new values?